CMakeLists.txt

```diff
@@ -352,6 +354,10 @@ install(
   FILES ${PROJECT_SOURCE_DIR}/vendor/ctpl_stl.h
   DESTINATION include/celerity/vendor
 )
+install(
+  FILES ${PROJECT_SOURCE_DIR}/vendor/small_vector/source/include/gch/small_vector.hpp
+  DESTINATION include/celerity/vendor/small_vector/source/include/gch
```

Comment:
Since we are in complete control of the install directory layout, we should consider shortening this path by stripping the `source` subdirectory.

Reply:
I was thinking about this but decided against it since I thought it would be less surprising to have the same layout as in the repo. Of course this was from our perspective rather than user perspective.

Reply:
Uses the same layout as if one were to install gch/small_vector now.